### ISSUE TITLE
Unicode Error when running in debug mode

### ISSUE THREAD

**Issue by apranav19** [mitsuhiko/flask#1611]
Hello,

I'm currently using flask while running `Python3.5` and I just noticed this `Unicode` error for the first time. When I run my app in the debug mode, I receive this error:

```
Traceback (most recent call last):
  File "main.py", line 28, in <module>
    app.run(debug=True)
  File "/Users/pranav/WebProjects/FitScout/fit_env/lib/python3.5/site-packages/flask/app.py", line 772, in run
    run_simple(host, port, self, **options)
  File "/Users/pranav/WebProjects/FitScout/fit_env/lib/python3.5/site-packages/werkzeug/serving.py", line 633, in run_simple
    application = DebuggedApplication(application, use_evalex)
  File "/Users/pranav/WebProjects/FitScout/fit_env/lib/python3.5/site-packages/werkzeug/debug/__init__.py", line 169, in __init__
    if self.pin is None:
  File "/Users/pranav/WebProjects/FitScout/fit_env/lib/python3.5/site-packages/werkzeug/debug/__init__.py", line 179, in _get_pin
    self._pin, self._pin_cookie = get_pin_and_cookie_name(self.app)
  File "/Users/pranav/WebProjects/FitScout/fit_env/lib/python3.5/site-packages/werkzeug/debug/__init__.py", line 96, in get_pin_and_cookie_name
    h.update('cookiesalt')
TypeError: Unicode-objects must be encoded before hashing
```

Is there an existing fix or do I need to change something on my end? I don't receive this error when running my app without the debug flag.

Thanks!

**Comment by apranav19** [comment]
Never mind, the same issue is referenced here: https://github.com/mitsuhiko/werkzeug/issues/798

Closing this issue now.

**Comment by mitsuhiko** [comment]
This is fixed in 0.11.1 of Werkzeug.
